files/checks/check_systemd_service.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+[ -n "$2" ] || exit 1;
+
+is_active=$(systemctl is-active "$2");
+echo "$2 is $is_active on $(hostname)"
+
+if [ "$is_active" == active ]; then
+	exit 0;
+else
+	exit 1;
```

Comment:
According to the [docs](https://nagios-plugins.org/doc/guidelines.html#AEN78) the correct return code here would be 2. You can't really distinguish between warning and cricital, so the default should be to return crit - or is there a specific reason here?